mlrun/api/crud/model_endpoints.py

```diff
@@ -4,6 +4,8 @@
 from nuclio.utils import DeployError
 from sqlalchemy.orm import Session
 from v3io.dataplane import RaiseForStatus
+from v3io_frames import frames_pb2 as fpb2
```

Comment:
"code is written once but read many times" - no one will understand what is `fpb2`, and in addition `frames_pb2` is not even that long
let's just use `frames_pb2` I don't see a reason for this shortcut